modules/appeals_api/app/controllers/appeals_api/v0/appeals_controller.rb

```diff
@@ -10,36 +10,16 @@ class AppealsController < ApplicationController
       skip_before_action(:authenticate)
 
       def index
-        log_request
         appeals_response = Caseflow::Service.new.get_appeals(
           target_veteran,
           'Consumer' => consumer,
           'VA-User' => requesting_va_user
         )
-        log_response(appeals_response)
-        render(
-          json: appeals_response.body
-        )
+        render(json: appeals_response.body)
       end
 
       private
 
-      def log_request
-        hashed_ssn = Digest::SHA2.hexdigest ssn
-        Rails.logger.info('Caseflow Request',
-                          'va_user' => requesting_va_user,
-                          'lookup_identifier' => hashed_ssn)
```

Comment:
Here I was thinking that since the PII is gone, the other log information is not that useful anymore, so I just got rid of all of it